packages/create-ship-app/scripts/backend.sh

```diff
@@ -3,17 +3,24 @@ set -e
 shopt -s dotglob
 
 project_name="$1"
-api_dir="$2"
+cli_dir="$2"
+api_dir="$3"
+api_type="$4"
+db_type="$5"
 
 mkdir "$project_name"
 cd "$project_name"
 
 git clone --quiet "https://github.com/paralect/ship"
 
-cp -a ship/services/$api_dir/ .
+cp -a ship/services/$api_dir/ api
```

Comment:
it will create next project strcture

```
/project-name
  /api
    / src
```
    
If someone creates only backend version of ship I don't think that we need to move api files in separate folder

I suppose next structure will be more convenient

```
/project-name
  /src
```